fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/ecam-cp.xml

```diff
@@ -443,6 +445,16 @@
             <SIMVAR>A32NX_BTN_MORE</SIMVAR>
         </UseTemplate>
 
+        <!-- ECAM ALL -->
+        <UseTemplate Name="FBW_Anim_Interactions">
+            <ANIM_TYPE>BUTTON_HELD</ANIM_TYPE>
+            <ANIM_TEMPLATE>FBW_ECAM_BUTTON_Template</ANIM_TEMPLATE>
+            <BASE_NAME>ALL</BASE_NAME>
+            <PART_ID>ECAM_ALL</PART_ID>
+            <TOOLTIPID>Cycle through ECAM pages</TOOLTIPID>
+            <SIMVAR>A32NX_BTN_ALL</SIMVAR>
```

Comment:
Would be a safer with a more descriptive prefix and more readable with a suffix describing the state, e.g. `A32NX_ECP_BTN_ALL_PRESSED`. The suffix is a suggestion, but the prefix identifying the system that writes the var I think should be required for all local vars.

Reply:
This is exactly the naming convention of the existing buttons and I will not start rewriting everything someone else has done. 
I was merely adding a missing button.

Reply:
Mmm, I wonder how we got these vars in in the first place, as they're badly named, but I suppose it makes sense to stick with it and fix in a different PR.